activerecord/lib/active_record/core.rb

```diff
@@ -239,9 +237,7 @@ def filter_attributes
       end
 
       # Specifies columns which shouldn't be exposed while calling +#inspect+.
-      def filter_attributes=(attributes_names)
-        @filter_attributes = attributes_names.map(&:to_s).to_set
```

Comment:
Can you remove `equire "set"` too?

https://github.com/rails/rails/blob/573ce860be319abcd036e555112a5f30b73db078/activerecord/lib/active_record/core.rb#L6